modules/schema/src/main/resources/lucuma/odb/graphql/OdbSchema.graphql

```diff
@@ -216,6 +216,107 @@ input ConstraintSetInput {
   elevationRange: ElevationRangeInput
 }
 
+
+
+
+
+
+
+
+scalar ChronicleId
+scalar TransactionId
+
+interface ChronicleEntry {
+  id: ChronicleId!
+  transactionId: TransactionId
+  user: User
+  timestamp: Timestamp!
+}
+
+type ConditionsEntry { # implements ChronicleEntry
```

Comment:
I think we can't do this kind of mapping yet. Need to ask Miles.

Reply:
There is a `SqlInterfaceMapping`, would that work?  Eg: https://github.com/gemini-hlsw/gsp-graphql/blob/main/modules/sql/shared/src/test/scala/SqlInterfacesMapping.scala

Reply:
The thing is, I don't need a discriminator for the insert result type. It's returning the concrete subtype so I don't know why extending the interface makes any difference to anything.